lib/spanner/bundle/config_validator.ex

```diff
@@ -51,6 +45,32 @@ defmodule Spanner.Bundle.ConfigValidator do
     validate_commands!(btype, commands)
   end
 
+  # Validate bundle Type
+  defp validate_bundle_type!(json, field) do
```

Comment:
My misunderstanding was that the embedded bundle used this code. I removed the code and tested to be sure. Thanks - definitely cleaner! 

I think the JsonNavigator code could still be refactored, but I don't think that belongs with this PR since it's not really related to removing the bundle type.